### PR TITLE
Issue11/add lic 4

### PR DESCRIPTION
**Pull request by angelicaengstrom**
My proposal for issue #11 "There exists at least one set of three consecutive data points that are the vertices of a triangle with area greater than AREA1", this includes
* functionality for LIC 4
* javadoc
* false and positive tests

Please review and suggest feedback

### CODE REVIEW

File: src/main/java/decide/LIC.java [hunk 1]
```diff
@@ -67,7 +88,7 @@ public boolean[] calculateCMV() {
         CMV[0] = LIC_0();
         CMV[1] = LIC_1();
         CMV[2] = LIC_2();
-        CMV[3] = LIC_3();
+        //CMV[3] = LIC_3();
```
Comment (tohanss) [inline src/main/java/decide/LIC.java:91]: Should not be commented out

**Review by tohanss — CHANGES_REQUESTED** [review]
Looks good, just the line that shouldnt be commented out. Just wait for CI branch to be merged and we can check if the tests pass.

**Review by TerenceNg03 — CHANGES_REQUESTED** [review]
Please merge CI from main first, so the check will run on this pr.

File: src/main/java/decide/LIC.java (hunk 1 above)
Comment (tohanss) [reply src/main/java/decide/LIC.java:91]: We can fix this with #29

**Review by TerenceNg03 — APPROVED** [review]

**Review by TerenceNg03 — CHANGES_REQUESTED** [review]
It seems that you commit `.idea` and `libraries` which should be ignored?

**Pull request merged**